src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -180,39 +162,11 @@ public String getEIBackendSubscriptionAddress() {
         LOG.debug("Input Request JSON Content to be forwarded:\n" + inputReqJsonContent);
         HttpEntity inputReqJsonEntity = new ByteArrayEntity(inputReqJsonContent.getBytes());
```

Comment:
Why removing the HTTP Return code from debug Message? Good to know when debugging the web-server.